halyard-core/src/test/groovy/com/netflix/spinnaker/halyard/core/resource/v1/LocalDiskProfileReaderSpec.groovy

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2017 Google, Inc.
```

Comment:
You should make this 2018, and your company or name

Reply:
Changed